tests/pytorch_tests/model_tests/feature_models/gptq_test.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import numpy as np
+import torch
+import torch.nn as nn
+from tests.pytorch_tests.model_tests.base_pytorch_feature_test import BasePytorchFeatureNetworkTest
+import model_compression_toolkit as mct
+from model_compression_toolkit.core.pytorch.default_framework_info import DEFAULT_PYTORCH_INFO
+from model_compression_toolkit.gptq.common.gptq_config import GradientPTQConfig, RoundingType
+from model_compression_toolkit.core.pytorch.utils import to_torch_tensor, torch_tensor_to_numpy
+from model_compression_toolkit.core.pytorch.utils import to_torch_tensor, torch_tensor_to_numpy
+from tests.common_tests.helpers.tensors_compare import normalized_mse, calc_db
+from model_compression_toolkit.gptq.pytorch.gptq_loss import multiple_tensors_mse_loss
+
+
+class TestModel(nn.Module):
+    def __init__(self):
+        super(TestModel, self).__init__()
+        self.conv1 = nn.Conv2d(3, 8, kernel_size=(3, 3), stride=(1, 1))
+        self.conv2 = nn.Conv2d(8, 16, kernel_size=(1, 1), stride=(1, 1))
+        self.activation = nn.SiLU()
+
+    def forward(self, inp):
+        x0 = self.conv1(inp)
+        x1 = self.activation(x0)
+        x2 = self.conv2(x1)
+        y = self.activation(x2)
+        return y
+
+
+class GPTQBaseTest(BasePytorchFeatureNetworkTest):
+    def __init__(self, unit_test):
+        super().__init__(unit_test, input_shape=(3, 16, 16))
+        self.seed = 0
+        self.experimental = True
+
+    def get_quantization_config(self):
+        return mct.QuantizationConfig(mct.QuantizationErrorMethod.NOCLIPPING, mct.QuantizationErrorMethod.NOCLIPPING)
+
+    def create_networks(self):
+        return TestModel()
+
+    def run_test(self):
+        # Create model
+        self.float_model = self.create_networks()
+
+        # Create quantization config
+        qConfig = self.get_quantization_config()
+
+        # Run MCT with PTQ
+        np.random.seed(self.seed)
+        ptq_model, ptq_qInfo = mct.pytorch_post_training_quantization_experimental(self.float_model,
```

Comment:
Please rename ptq_qInfo to follow conventions (maybe ptq_user_info or something like that)